delta/plugins/storage/src/test/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/storages/operations/remote/RemoteDiskStorageAccessSpec.scala

```diff
@@ -0,0 +1,53 @@
+package ch.epfl.bluebrain.nexus.delta.plugins.storage.storages.operations.remote
+
+import akka.actor.ActorSystem
+import akka.testkit.TestKit
+import ch.epfl.bluebrain.nexus.delta.plugins.storage.storages.model.StorageRejection.StorageNotAccessible
+import ch.epfl.bluebrain.nexus.delta.plugins.storage.storages.model.StorageValue.RemoteDiskStorageValue
+import ch.epfl.bluebrain.nexus.delta.plugins.storage.storages.operations.remote.RemoteStorageDocker.{RemoteStorageServicePort, _}
+import ch.epfl.bluebrain.nexus.delta.plugins.storage.storages.permissions._
+import ch.epfl.bluebrain.nexus.delta.sdk.model.{BaseUri, Label}
+import ch.epfl.bluebrain.nexus.delta.sdk.syntax._
+import ch.epfl.bluebrain.nexus.testkit.{IOValues, TestHelpers}
+import monix.execution.Scheduler
+import org.scalatest.DoNotDiscover
+import org.scalatest.concurrent.Eventually
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpecLike
+
+@DoNotDiscover
```

Comment:
To delete no ?

Reply:
No. This is correct, since all the suite gets called from `RemoteStorageSpec`